openlibrary/templates/type/author/view.html

```diff
@@ -123,6 +123,11 @@ <h2 class="author collapse">
 
     <div class="contentTwothird" style="margin-bottom:0;">
 
+        <div class="illustration author-image">
```

Comment:
When you duplicate identical elements in the HTML, it's usually a sign that the initial design is bad from a mobile perspective. I do believe search engines have been known to give negative penalties for duplicate HTML markup.

It might be worth spending some time prototyping whether rearranging the HTML, or using CSS can achieve the same effect. Have you explored using display: flex and order ?

Reply:
I completely agree with you the image should be positioned below the description instead of positioning it below 100s of works.
I think I found a solution of achieving the same with flex and order.
Mobile View:
![image](https://user-images.githubusercontent.com/64412143/120880097-d49a5980-c5e5-11eb-8c13-8baed3650f19.png)
Desktop View:
![image](https://user-images.githubusercontent.com/64412143/120880197-630edb00-c5e6-11eb-8806-5311ae3ffdd3.png)